gatsby-config.js

```diff
@@ -93,5 +92,16 @@ module.exports = {
                 policy: [{ userAgent: '*', allow: '/' }],
             },
         },
+        {
+            resolve: `gatsby-plugin-netlify`,
+            options: {
+                headers: {
+                    '/page-data/*': [
+                        'cache-control: max-age=0,no-cache,no-store,must-revalidate',
```

Comment:
I think just `must-revalidate` should do to prevent cache invalidation, the cache is still needed for faster performance right? and limit max-age, probably like 1 week or 1 month